src/hazelcore/Var.re

```diff
@@ -25,10 +25,14 @@ let is_false = eq("false");
 
 let is_let = eq("let");
 
+let is_fun = eq("fun");
+
 let is_case = eq("case");
 
 let is_wild = eq("_");
 
+let is_keyword = s => is_let(s) || is_fun(s) || is_case(s);
```

Comment:
Let's change this to use ExpandingKeyword.mk -- same goal, to have it so we don't need to find multiple places in the codebase to change when a new keyword is added